layer1/bridge/consensus/main.mligo

```diff
@@ -0,0 +1,33 @@
+#import "errors.mligo" "Error"
+#import "parameter.mligo" "Parameter"
+#import "update_root_hash.mligo" "Update_root_hash"
+#import "deposit.mligo" "Deposit"
+#import "withdraw.mligo" "Withdraw"
+
+type storage = {
```

Comment:
Can you create a storage.mligo module with appropriate types
exemple https://github.com/ligolang/randomness-cameligo/blob/main/contracts/storage.mligo

Reply:
@oteku I noticed this `storage.mligo` in this example. However, I think that putting the source code of `deposit.mligo`, `withdraw.mligo`, and `update_root_hash.mligo` modules inside `storage.mligo` module makes it complicated. Therefore, I think I can create `storage.mligo` and this module calls the `deposit.mligo`, `withdraw.mligo`, and `update_root_hash.mligo` sub-modules.